OpenOversight/app/templates/list_officer.html

```diff
@@ -145,13 +149,13 @@ <h3 class="panel-title accordion-toggle">Age range</h3>
               <div class="col-md-6 col-xs-12">
                 <h2>
                   <a href="{{ url_for('main.officer_profile', officer_id=officer.id) }}" id="officer-profile-{{ loop.index }}">{% include "partials/officer_name.html" %}</a>
-                  <small>#{{ assignment.star_no }}</small>
+                  <small>{{ star_no }}</small>
```

Comment:
If the `{% if officer.assignments.all() %}` clause evals as false, `star_no` won't get set so won't this throw an error? If so it might need to change to `<small>{{ star_no|default('') }}</small>`.

Reply:
Nevermind, I tried it out and it didn't throw an error.